utilities/implementation-kit/data-migration/create_user_bihar.py

```
@@ -0,0 +1,224 @@
+
+
+def connect():
+    accesstoken = accessToken()
+    print(accesstoken)
+    wrkbk = \
+        openpyxl.load_workbook(path)
+
+    sh = wrkbk.active
+    if os.path.exists("sms.txt"):
+        os.remove("sms.txt")
+    file = open("sms.txt", "a", encoding="utf-8")
```

Ruff

Use a context manager for opening files

(SIM115)